nvflare/job_config/fed_job.py

```diff
@@ -260,19 +260,17 @@ def to(
             else:  # TODO: handle other persistors
                 added_model = False
                 # Check different models framework types and add corresponding persistor
-                if torch_ok:
-                    if isinstance(obj, nn.Module):  # if model, create a PT persistor
-                        component = PTFileModelPersistor(model=obj)
-                        self._deploy_map[target].app.add_component("persistor", component)
-
-                        component = PTFileModelLocator(pt_persistor_id="persistor")
-                        self._deploy_map[target].app.add_component("model_locator", component)
-                        added_model = True
-                elif tf_ok:
-                    if isinstance(obj, tf.keras.Model):  # if model, create a TF persistor
-                        component = TFModelPersistor(model=obj)
-                        self._deploy_map[target].app.add_component("persistor", component)
-                        added_model = True
+                if torch_ok and isinstance(obj, nn.Module):  # if model, create a PT persistor
```

Comment:
What is this change for? The old and new code seem to be identical, other than different format.

Reply:
In the old version, if torch is installed (`torch_ok=True`) but the model is a `tf.keras.Model`, it will not reach the second case, and won't add the TFModelPersistor.

Reply:
I see.